src/terraform/command_helpers.go

```diff
@@ -148,3 +135,8 @@ func getDependencyServiceEnvVars(deployConfig deploy.Config, serviceConfig types
 	}
 	return result
 }
+func extractDependencyData(dependencyName string, deployConfig deploy.Config) (string, error) {
```

Comment:
Can we name this `getDependencyServiceData`? (and add a newline above it so we have en empty line between functions). Feels weird that the return value of `extractDependencyData` is assigned to serviceData